content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -124,38 +132,56 @@ based on formulas.
 Check out the :ref:`Advanced price rules (discounts, formulas) <sales/advanced-price-rules>` section
 for detailed steps on how to add advanced price rules to a pricelist.
 
-Time-based rules tab
+Recurring Prices tab
 --------------------
 
-Time-based rules are used specifically with :doc:`subscription products
-</applications/sales/subscriptions/products>`. Be sure to check out the Odoo *Subscriptions*
-:doc:`documentation </applications/sales/subscriptions>`.
+Recurring prices are specifically used with :doc:`subscription products
+</applications/sales/subscriptions/products>`.
 
-Under the :guilabel:`Time-based rules` tab, the same functionality of the :guilabel:`Price Rules`
-tab is present, with the only difference being that a repeating time period can be applied in the
-:guilabel:`Period` column.
+.. seealso::
+   :doc:`/applications/sales/subscriptions`
 
```

Comment:
Seems unnecessary to have a seealso to the same Subscriptions doc at the top and bottom of this section. Suggesting to remove this one and use the seealso at the bottom of the section 
```suggestion
```